selfdrive/controls/radard.py

```diff
@@ -158,23 +164,25 @@ def get_RadarState_from_vision(lead_msg: capnp._DynamicStructReader, v_ego: floa
     "aLeadTau": 0.3,
     "fcw": False,
     "modelProb": float(lead_msg.prob),
+    "status": True,
     "radar": False,
-    "status": True
+    "radarTrackId": -1,
   }
 
 
 def get_lead(v_ego: float, ready: bool, tracks: Dict[int, Track], lead_msg: capnp._DynamicStructReader,
              model_v_ego: float, low_speed_override: bool = True) -> Dict[str, Any]:
   # Determine leads, this is where the essential logic happens
-  if len(tracks) > 0 and ready and lead_msg.prob > .5:
+  lead_msg_empty = any([lead_msg.x, lead_msg.y, lead_msg.v])
+  if len(tracks) > 0 and ready and not lead_msg_empty and lead_msg.prob > .5:
```

Comment:
what does this do?

Reply:
i think this was for fuzzy tests to pass - fuzzy test would create leads with x/y/v of variable length